workflow/scripts/postprocess_tpm.py

```
@@ -0,0 +1,21 @@
+import sys
+import pandas as pd
+
+sys.stderr = open(snakemake.log[0], "w")
+
+# Load input files
+tpm_df = pd.read_csv(snakemake.input["tpm"], sep="\t")
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)